HHS/OPRE-OPS
# feat: :sparkles: adds Accordion component

## What changed
feat: :sparkles: adds Accordion component
- optional heading level. defaults to h4. keyboard accessible.

## Issue

#1546 

## How to test

1. add component
2. supply props
3. see of it opens and closes via mouse click or keyboard spacebar

## Screenshots

https://github.com/HHS/OPRE-OPS/assets/4629398/31da8081-a019-43a2-833e-ee98baf3d19c

## Definition of Done Checklist
